packages/errors/src/message-formatter.ts

```diff
@@ -16,6 +16,10 @@ function encodeValue(value: unknown): string {
         );
     } else if (typeof value === 'bigint') {
         return `${value}n`;
+    } else if (value === null) {
+        return 'null';
+    } else if (value === undefined) {
+        return 'undefined';
```

Comment:
These special cases are required because the `else` block calls `Object.getPrototypeOf(value)` which is a type error for these values:

```node
> Object.getPrototypeOf(null)
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
> Object.getPrototypeOf(undefined)
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
```

I was hitting this in my instructions test when it throws the error with undefined accounts/data.

Reply:
I believe I have a fix for this in my stack with just `value != null`.

Reply:
[packages/errors/src/message-formatter.ts](https://github.com/solana-labs/solana-web3.js/blob/6cfd2d2d66972626c08f67e87c6ecfb8077c3111/packages/errors/src/message-formatter.ts#L22)

Reply:
Clever! TIL `undefined != null`. Any idea what else that catches?

Reply:
I originally had `value != null && value != undefined` but Steve pointed out that `!=` will actually evaluate both, as compared to `!==`. Seems it's because an `undefined` value with a single equals will evaluate the conditional as "falsy".

Reply:
Removed, Steve merged #2223 separately